bundles/org.eclipse.ui.themes/css/e4_preview_gtk.css

```
@@ -33,11 +33,11 @@ ColorDefinition#org-eclipse-ui-workbench-ACTIVE_TAB_OUTLINE_COLOR {
 }
 
 ColorDefinition#org-eclipse-ui-workbench-INACTIVE_UNSELECTED_TABS_COLOR_START {
-	color: #F8F8F8;
+	color: #F8F8F8; /* same as '#org-eclipse-ui-workbench-SECONDARY_BACKGROUND' cannot use '#org-eclipse-ui-workbench-SECONDARY_BACKGROUND' directly*/
```

maybe shorten this a bit to not have to mention it twice:
```suggestion
	color: #F8F8F8; /* same as '#org-eclipse-ui-workbench-SECONDARY_BACKGROUND' cannot reference it directly*/
```

Is this something we can improve technically?

---

> maybe shorten this a bit to not have to mention it twice:
> 
> Is this something we can improve technically?

I am not sure. I didn't yet have a look into it.